main.go

```diff
@@ -270,7 +270,7 @@ func validateVargs(vargs *GAE) error {
 	}
 
 	if vargs.AppCfgCmd == "" {
-		vargs.AppCfgCmd = "/go_appengine/appcfg.py"
+		vargs.AppCfgCmd = "/google-cloud-sdk/platform/google_appengine/appcfg.py"
```

Comment:
Assuming appcfg is not in the path?

Reply:
correct - gcloud proper is in the path, but this utility is now